inference-engine/src/mkldnn_plugin/nodes/mkldnn_color_convert_node.cpp

```diff
@@ -0,0 +1,1113 @@
+
+    void execute(mkldnn::stream strm) override {
+        const auto & kernel = jit_converter_create<T>();
+        const auto & dims = inputDims(0);
+
+        const size_t batch_size = dims[N_DIM];
+        const size_t height = dims[H_DIM] * 2 / 3;
+        const size_t width = dims[W_DIM];
+
+        const T* y = static_cast<const T*>(input(0));
+        const T* u = y + width * height;
+        const T* v = y + 5 * width * height / 4;
+        T* dst = static_cast<T*>(output(0));
+
+        const size_t stride_y = height * width * 3 / 2;
+        const size_t stride_uv = height * width * 3 / 2;
+
+        InferenceEngine::parallel_for2d(batch_size, height, [&](int batch, int h) {
+            typename jit_uni_converter::Params args;
+            args.y = y + batch * stride_y + h * width;
+            args.u = u + batch * stride_uv + (h / 2) * (width / 2);
+            args.v = v + batch * stride_uv + (h / 2) * (width / 2);
+            args.dst = dst + (batch * width * height + h * width) * 3;
+            args.width = width;
+            args.colorFormat = _colorFormat[0]; // The first byte is enough to determine the RGB or BGR format.
+            kernel(args);
+        });
+    }
+};
+
+template<typename T>
+class ThreePlaneConvert<T, impl_desc_type::jit_uni> : public Converter {
+public:
+    using Converter::Converter;
+
+    void execute(mkldnn::stream strm) override {
+        const auto & kernel = jit_converter_create<T>();
+        const auto & dims = inputDims(0);
+
+        const T* y = static_cast<const T*>(input(0));
+        const T* u = static_cast<const T*>(input(1));
+        const T* v = static_cast<const T*>(input(2));
+        T* dst = static_cast<T*>(output(0));
+
+        const size_t batch_size = dims[N_DIM];
+        const size_t height = dims[H_DIM];
+        const size_t width = dims[W_DIM];
+
+        const size_t stride_y = height * width;
+        const size_t stride_uv = height * width / 4;
+
+        InferenceEngine::parallel_for2d(batch_size, height, [&](int batch, int h) {
+            typename jit_uni_converter::Params args;
+            args.y = y + batch * stride_y + h * width;
+            args.u = u + batch * stride_uv + (h / 2) * (width / 2);
+            args.v = v + batch * stride_uv + (h / 2) * (width / 2);
+            args.dst = dst + (batch * width * height + h * width) * 3;
+            args.width = width;
+            args.colorFormat = _colorFormat[0]; // The first byte is enough to determine the RGB or BGR format.
+            kernel(args);
+        });
+    }
+};
+
+}   // namespace i420
+
+}   // namespace
+
+MKLDNNColorConvertNode::Converter::Converter(MKLDNNNode *node, const ColorFormat & colorFormat)
+    : _node(node)
+    , _colorFormat(colorFormat) {
+}
+
+InferenceEngine::Precision MKLDNNColorConvertNode::Converter::inputPrecision(size_t idx) const {
+    return _node->getParentEdgeAt(idx)->getMemory().GetDescWithType<BlockedMemoryDesc>()->getPrecision();
```

Comment:
better to use `get*EdgesAtPort`
it's necessary for child edges when node has several output port
unlikely `ColorConvert` will have more that one output ports but better avoid it in advance

also it's not necessary to call `GetDescWithType` to get precision enough `getDesc()`

Reply:
getDesc() is private. So I can't use it.

Reply:
@vladislav-volkov It is definitely public: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/mkldnn_memory.h#L52. So lets refactor this

Reply:
done